pkg/didcomm/packer/authcrypt/pack_test.go

```diff
@@ -79,8 +79,8 @@ func TestAuthryptPackerUsingKeysWithDifferentCurvesSuccess(t *testing.T) {
 	_, recipientsKey1, keyHandles1 := createRecipients(t, k, 1)
 	// since authcrypt does ECDH kw using the sender key, the recipient keys must be on the same curve as the sender's.
```

Comment:
Looks like you need to update this comment now

Reply:
sure.. I'll update..

Reply:
updated